io/disk/tiobench.py.data/tiobench.yaml

```diff
@@ -0,0 +1,33 @@
+setup: 
+    location: !mux
+        default:
+            target: '/mnt'
+
+    block: !mux
+        first:
+            blocks: 4096
+        second:
+            blocks: 8192
+
+    thread: !mux
+        default:
+            threads: 10
+        minimal:
+            threads: 5
+
+    blocksize: !mux
+        default: 
+            size: 1024
+     
+    runs: !mux
+        default: 
+            numruns: 2
+        minimal:
+            numruns: 1
+  
+    iteration: !mux
```

Comment:
So, After this change, will this be accessible in a variable like `self.iteration` for usage in code?
And for now if we run it with this,
```
    iteration: !mux
        default:
        quick:
            1:
            2:
```
Is there a way to get the iteration number?